src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,115 @@
+            if (resultSet.next()) {
+                manufacturer = new Manufacturer();
+                manufacturer.setId(resultSet.getObject("id", Long.class));
+                manufacturer.setName(resultSet.getString("name"));
+                manufacturer.setCountry(resultSet.getString("country"));
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't get manufacturer from DB id:" + id, e);
+        }
+        return Optional.ofNullable(manufacturer);
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        String getAllRequest = "SELECT * FROM manufactures where is_deleted = false";
+        List<Manufacturer> allManufacturers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                 Statement getAllManufacturesStatement = connection.createStatement()) {
+            ResultSet resultSet = getAllManufacturesStatement
+                    .executeQuery(getAllRequest);
+            while (resultSet.next()) {
+                Long id = resultSet.getObject("id", Long.class);
+                String name = resultSet.getString("name");
+                String country = resultSet.getString("country");
+                Manufacturer manufacturer = new Manufacturer();
+                manufacturer.setCountry(country);
+                manufacturer.setId(id);
+                manufacturer.setName(name);
+                allManufacturers.add(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all manufacturers from DB", e);
+        }
+        return allManufacturers;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateRequest = "UPDATE manufactures SET name = ?, country = ?"
+                + "where id = ? AND is_deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement createdManufacturerStatement =
+                         connection.prepareStatement(updateRequest)) {
+            createdManufacturerStatement.setString(1, manufacturer.getName());
+            createdManufacturerStatement.setString(2, manufacturer.getCountry());
+            createdManufacturerStatement.setLong(3, manufacturer.getId());
+            createdManufacturerStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert manufacturer from DB", e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteRequest = "UPDATE manufactures SET is_deleted = true where id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createdManufacturerStatement =
+                        connection.prepareStatement(deleteRequest,
+                                Statement.RETURN_GENERATED_KEYS)) {
```

Comment:
```suggestion
                        connection.prepareStatement(deleteRequest)) {
```
You delete data. You don't need take the key.